lib/src/base-styles/_nx-colors.scss

```diff
@@ -76,5 +80,9 @@
     //NxBinaryDonutChart variables
     --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-30);
     --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-pink-80);
+
+    // NxGlobalHeader
+    --nx-color-global-header-background: var(--nx-swatch-indigo-20);
+    --nx-color-global-header-border: var(--nx-swatch-indigo-15);
```

Comment:
From the way the notes on the mockup are worded it seems to me like it is not a coincidence that the header border matches the page background in dark mode.  As such, I think `--nx-color-site-background` should be used here instead of using the swatch directly.

Reply:
> From the way the notes on the mockup are worded it seems to me like it is not a coincidence that the header border matches the page background in dark mode. As such, I think `--nx-color-site-background` should be used here instead of using the swatch directly.

Ah I see, I fixed that here: ef41aa01e135d07b51bbcd85a6470c8590920ab8